docs/UserGuide.md

```diff
@@ -3,271 +3,478 @@ layout: page
-* (example of a client with insurance policy and URL)`add n/Tom Tan p/91234567 e/tomt@example.com a/Orchard i/Policy_1274>www.myinsurancecompany.com/policy_1274 t/basic`
-* (example with no insurance policy and no tag)`add n/Betsy Crowe e/betsycrowe@example.com a/Kent Ridge`
+**Examples**:
+* Example of a client with insurance policy but no URL associated with insurance policy
+    * `add n/John Doe p/98765432 e/johnd@email.com a/Orchard i/Pol_#123456 t/basic`
+* Example of a client with insurance policy and URL associated with insurance policy
+    * `add n/Tom Tan p/91234567 e/tomt@email.com a/Orchard i/Policy_1274>www.myinsurancecompany.com/policy_1274 t/basic`
+* Example with no insurance policy and no tag
+    * `add n/Betsy Crowe e/betsycrowe@email.com a/Kent Ridge`
 
-### Listing all persons : `list [-ATTRIBUTE]`
+[Return to Table of Contents](#table-of-contents)
 
-![list_with_filter](images/list_with_filter.png)
+### Edit client contact: `edit`
 
-Shows a list of all clients in ClientBook. An optional attribute option can be added to show a list with only the specified attribute.
+**Purpose**: Edits an existing client contact in the ClientBook.
 
-Format: `list [-ATTRIBUTE]…​`
+**Format**: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [i/POLICY_NUMBER] [t/TAG]…​`
 
-Examples:
-* `list` Shows a list of all clients and all their information
-* `list -policy` Shows a list of all clients and their policy numbers
-* `list -phone -policy` Shows a list of all clients, their phone number, and their policy numbers
+* Edits the client at the specified `INDEX`.
+    * The index refers to the index number shown in the displayed client list.
+    * The index must be more than 1, and less than or equal to the index of the last item in the displayed list.
+* At least one of the optional fields must be provided.
 
-### Editing a person : `edit`
+:exclamation: **Caution**: Existing values will be **replaced** with the input values.
 
-Edits an existing client in the ClientBook.
+**Examples**:
+*  Edit the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
+    * `edit 1 p/91234567 e/johndoe@example.com`
+      
+      ![edit 1](images/edit-1.png)
+    
+*  Edit the name of the 2nd person to be `Betsy Crower`.
+    * `edit 2 n/Betsy Crower`
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [i/POLICY_NUMBER] [t/TAG]…​`
+[Return to Table of Contents](#table-of-contents)
 
-* Edits the client at the specified INDEX. The index refers to the index number shown in the displayed client list. The index must be a positive integer 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
+### List all clients : `list`
+
+**Purpose**: Shows a list of all clients in ClientBook. An optional attribute option can be added to show a list with only the specified attribute.
```

Comment:
This line suggests that only 1 optional attribute can be added, while one of the example below shows multiple optional attributes being used. This can be confusing to the reader.